scripts/Muon/GUI/ElementalAnalysis2/periodic_table_tab/periodic_table_tab_presenter.py

```diff
@@ -0,0 +1,248 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
are these just copies of the original?

Reply:
Alot of the code was copied from https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py
but some new feautures were added

Reply:
you should inherit the code instead and add new functionality in the derived class